roles/wordpress-setup/tasks/nginx-includes.yml

```diff
@@ -13,7 +13,30 @@
       recurse: yes
     become: no
     delegate_to: localhost
-    register: nginx_includes_templates
+    register: nginx_includes_templates_all
+
+  - name: Normalize extra include folders to a list
+    set_fact:
+      nginx_includes_extra_folders_list: >-
+        {{
+          nginx_includes_extra_folders
+          if nginx_includes_extra_folders is iterable and not nginx_includes_extra_folders is string
+          else [nginx_includes_extra_folders | default('all')]
+        }}
+
+  - name: Combine site keys with extra include folders
+    set_fact:
+      nginx_include_contexts: "{{ wordpress_sites.keys() | list + nginx_includes_extra_folders_list }}"
+
+  - name: Filter Nginx includes templates by existing sites and extra folders
+    set_fact:
+      nginx_includes_templates:
+        files: >-
+          {{
+            nginx_includes_templates_all.files
+            | selectattr('path', 'search', '/(' + nginx_include_contexts | join('|') + ')/')
+            | list
+          }}
 
```

Comment:
Could we use `nginx_include_contexts` to build the patterns for use in the `find`? Basically, move the "Build list of Nginx includes templates" task _after_ this one, and make it look something like this:

```yml
- name: Build list of Nginx includes templates
  find:
    paths: "{{ nginx_includes_templates_path }}"
    patterns: "{{ [item + '/*.conf.j2' for item in nginx_include_contexts] }}"
    recurse: yes
  become: no
  delegate_to: localhost
```

Reply:
Yeah that'd be cleaner. Thanks! Will review and push changes up 👍

Reply:
>```yaml
>patterns: "{{ [item + '/*.conf.j2' for item in nginx_include_contexts] }}"
>```

This caused an error in testing:

```bash
TASK [wordpress-setup : Build list of Nginx includes templates] *******************************************************************************************************************
fatal: [staging1 -> localhost]: FAILED! => {"msg": "template error while templating string: expected token ',', got 'for'. String: {{ [item + '/*.conf.j2' for item in nginx_include_contexts] }}. expected token ',', got 'for'"}
```

Swapped for 
```yaml
patterns: "{{ nginx_include_contexts | map('regex_replace', '^(.*)$', '\\1/*.conf.j2') | list }}"
```

Reply:
Looks good 👍